maturity-level-one/README.md

```diff
@@ -143,6 +143,7 @@ This allows you to be secure on the physical API while the consumers can still u
     - Less details compared to a custom data contract
     - Read [this blog post](https://tech.domain.com.au/2017/11/please-dont-spare-me-the-details/) on how to achieve this
   - If the above suggestion is not possible you should use a custom data contract. See Microsoft example [here](https://github.com/Microsoft/api-guidelines/blob/master/Guidelines.md#7102-error-condition-responses).
+- For all controllers, document all possible return status codes using a [SwaggerResponse] attribute.
```

Comment:
Can you put this comment in the "Generating OpenAPI documentation" table ?
There is a row with SwaggerResponse attribute

Reply:
Fixed